paradise.dme

```diff
@@ -1447,6 +1447,7 @@
 #include "code\modules\events\vent_clog.dm"
 #include "code\modules\events\wallrot.dm"
 #include "code\modules\events\wormholes.dm"
+#include "code\modules\events\holidays\Christmas.dm"
```

Comment:
I'm 99% sure that this file is obsolete and was just kept around for goodness knows why. There is a very similar file at `code\modules\holiday\christmas.dm`